BE/GovernmentEntities/EuropeanJurisdiction/EasternEuropeGovernmentEntitiesAndJurisdictions.rdf

```diff
@@ -1,339 +1,339 @@
+		<fibo-be-ge-ge:hasFullSovereigntyOver rdf:resource="&lcc-3166-1;Bulgaria"/>
+		<fibo-be-ge-ge:isRepresentedBy rdf:resource="&fibo-be-ge-eeuj;GovernmentOfTheRepublicOfBulgaria"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-ge-eeuj;CzechEntity">
+		<rdf:type rdf:resource="&fibo-be-ge-ge;SovereignState"/>
+		<rdfs:label xml:lang="en">Czech entity</rdfs:label>
+		<rdfs:label xml:lang="cs">Český subjekt</rdfs:label>
+		<skos:definition>sovereign state and polity that is Czechia</skos:definition>
+		<fibo-be-ge-ge:hasFullSovereigntyOver rdf:resource="&lcc-3166-1;Czechia"/>
+		<fibo-be-ge-ge:isRepresentedBy rdf:resource="&fibo-be-ge-eeuj;GovernmentOfTheCzechRepublic"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-ge-eeuj;GovernmentOfHungary">
+		<rdf:type rdf:resource="&fibo-be-ge-ge;NationalGovernment"/>
+		<rdfs:label xml:lang="en">Government of Hungary</rdfs:label>
+		<rdfs:label xml:lang="hu">Magyarország kormánya</rdfs:label>
+		<rdfs:seeAlso rdf:resource="https://www.kormany.hu/en/"/>
+		<skos:definition>unitary, dominant-party, parliamentary republic, bordering Slovakia to the north, Ukraine to the northeast, Romania to the east and southeast, Serbia to the south, Croatia and Slovenia to the southwest, and Austria to the west</skos:definition>
+		<fibo-be-ge-ge:hasJurisdiction rdf:resource="&fibo-be-ge-eeuj;JurisdictionOfHungary"/>
+		<fibo-fnd-rel-rel:governs rdf:resource="&lcc-3166-1;Hungary"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-ge-eeuj;GovernmentOfRomania">
+		<rdf:type rdf:resource="&fibo-be-ge-ge;NationalGovernment"/>
+		<rdfs:label xml:lang="en">Government of Romania</rdfs:label>
+		<rdfs:label xml:lang="ro">Guvernul României</rdfs:label>
+		<rdfs:seeAlso rdf:resource="https://www.gov.ro/en/"/>
+		<skos:definition>unitary, semi-presidential republic at the crossroads of Central, Eastern, and Southeastern Europe, bordering Bulgaria to the south, Ukraine to the north, Hungary to the west, Serbia to the southwest, and Moldova to the east</skos:definition>
+		<fibo-be-ge-ge:hasJurisdiction rdf:resource="&fibo-be-ge-eeuj;JurisdictionOfRomania"/>
+		<fibo-fnd-rel-rel:governs rdf:resource="&lcc-3166-1;Romania"/>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-be-ge-eeuj;GovernmentOfTheCzechRepublic">
+		<rdf:type rdf:resource="&fibo-be-ge-ge;NationalGovernment"/>
+		<rdfs:label xml:lang="en">Government of the Czech Republic</rdfs:label>
+		<rdfs:label xml:lang="cs">Vláda České republiky</rdfs:label>
+		<rdfs:seeAlso rdf:resource="https://www.vlada.cz/en/"/>
+		<skos:definition>unitary, parliamentary, constitutional republic and multi-party, representative democracy, with the president as head of state and prime minister as head of government, that is a landlocked country in Central Europe bordered by Austria to the south, Germany to the west, Poland to the northeast and Slovakia to the southeast</skos:definition>
```

Comment:
this definition goes beyond the Government and describes the geography of the country. Further, the definition relies on incidental non-definitional aspects. e.g. if Poland were renamed Polska, or even merged with Slovakia , that would not change the essence of Czechia.